### ISSUE TITLE
Feature request: Start REPL in Editor Area to the side

### ISSUE THREAD

**Issue by juandarias** [julia-vscode/julia-vscode#2922]
VS Code has for a while the command `Create a New Terminal in Editor Area to the side`. When developing, I like to have the REPL in this editor area instead of the standard terminal location (horizontal or vertical panel), as I can make better use of my screen area and create the layout that fits the best for my work.  See the screenshot for an example.

![layout](https://user-images.githubusercontent.com/31775008/173866068-cc0cf4c7-4cab-4d1b-a4e1-2b8336d3b167.png)

At this moment, the only way of getting the REPL into the desired location requires two commands plus having to close the panel which remains open after moving the terminal into the editor area.

It would be great if a command such as `Start REPL in Editor Area to the side` could be made available.

Thanks for the great extension!

**Comment by pfitzseb** [comment]
I'd like to solve this by opening the Julia terminal wherever the user had it before (in a pane or specific view column), but that depends on https://github.com/microsoft/vscode/issues/152302.

**Comment by pfitzseb** [comment]
Well, kinda sorta but not really. I'm also not super convinced that we actually want to move away from the "one blessed terminal" paradigm for now -- it's just really nice UX imho.

**Comment by davidanthoff** [comment]
Yeah, I think if we enable multiple REPLs, we should make it hard ;) Or at least such that one _really_ must want to leave the one-REPL situation.
